lib/rules/css-orientation-lock.json

```diff
@@ -0,0 +1,19 @@
+{
+	"id": "css-orientation-lock",
+	"selector": "html",
+	"tags": [
+		"cat.structure",
+		"wcag262",
+		"wcag21aa"
+	],
+	"metadata": {
+		"description": "Ensures content is not locked to any specific display orientation, and the content is operable in all display orientations",
+		"help": "Content is operable in all display orientations"
```

Comment:
I think this needs to be more generic. Something along the lines of "CSS Media queries are not used to lock display orientation"?